Makefile

```diff
@@ -36,6 +36,7 @@ GOBIN ?= /work/out/bin
 RUN = docker run -t --sig-proxy=true -u $(UID) --rm \
 	-v /etc/passwd:/etc/passwd:ro \
 	-v $(readlink /etc/localtime):/etc/localtime:ro \
+	-v $(GOPATH)/src/istio.io/operator:/go/src/istio.io/operator \
```

Comment:
this will be overwritten when `make common` is run to update the makefiles, which may be often.

Reply:
I edited the above comment to indicate the correct make target that results in an overwrite of the Makefile.

Reply:
ok I will move this logic to core instead